src/index.css

```diff
@@ -50,4 +50,30 @@ h5 {
 
 input[type = 'text'] {
   @apply text-xl rounded-3xl border-2 border-[#AAAAAA] bg-[#EEEEEE] w-full px-4 py-1;
+}
+
+textarea[type = 'textarea'] {
+  @apply text-xl rounded-3xl border-2 border-[#AAAAAA] bg-[#EEEEEE] w-full px-4 py-1;
+}
+
+select{
+  @apply text-xl rounded-3xl border-2 border-[#AAAAAA] bg-[#EEEEEE] w-full px-4 py-1;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+}
```

Comment:
The clickable portion doesn't extend to the end of the box which is confusing when there is an arrow.